neworigins/extra.cpp

```diff
@@ -128,10 +129,59 @@ static void CreateQuest(ARegionList *regions, int monfaction)
 
 	q = new Quest;
 	q->type = -1;
-	item = new Item;
-	item->type = I_RELICOFGRACE;
-	item->num = 1;
-	q->rewards.Add(item);
+
+	// Set up quest rewards
+	count = 0;
```

Comment:
copied this idea from here https://github.com/artyomtrityak/Atlantis/blob/master/army.cpp#L683